tools/assetpipelineeditor/exportdialog.ui

```diff
@@ -0,0 +1,314 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
could you remove the enabled and make the `QGroupBox` checkable?

Reply:
done